pyproject.toml

```diff
@@ -0,0 +1,27 @@
+[tool.poetry]
+name = "desec-dns"
+version = "0.1.0"
```

Comment:
Leave the version at `0.0.0` for now. I'd like to get it from git tags in the future.